ignore the ack delay in Initial and Handshake packets (#2676)
quicwg/base-drafts: draft-ietf-quic-recovery.md

samples, and rttvar is the endpoint's estimated variance in the RTT samples.

> smoothed_rtt uses path latency after adjusting RTT samples for peer-reported

Reviewer: ```suggestion
The calculation of smoothed_rtt uses path latency after adjusting RTT samples for```

host delays ({{host-delay}}).  For packets sent in the ApplicationData packet
number space, a peer limits any delay in sending an acknowledgement for an

> ack-eliciting packet to no greater than the advertised max_ack_delay transport

Reviewer: ```suggestion
ack-eliciting packet to no greater than the value it advertises in the max_ack_delay transport
```
